.github/workflows/build-future.yml

```diff
@@ -34,7 +34,7 @@ jobs:
       xdr_ref: v20.0.2
       core_ref: v20.1.0
```

Comment:
Should we wait for core [quickstart PR](https://github.com/stellar/quickstart/pull/593/files) to merge prior to this?

Reply:
yes, I've got this in draft, so we can keep it parked until ready, thanks!